hummingbot/market/radar_relay/radar_relay_market.pyx

```diff
@@ -253,22 +255,22 @@ cdef class RadarRelayMarket(MarketBase):
                 )
                 await asyncio.sleep(0.5)
 
-    cdef object c_get_fee(self,
-                          str base_currency,
-                          str quote_currency,
-                          object order_type,
-                          object trade_type,
-                          object amount,
-                          object price):
-        cdef:
-            int gas_estimate = 130000  # approximate gas used for 0x market orders
-            double transaction_cost_eth
+    @staticmethod
+    def c_get_fee(base_currency: str,
+            str quote_currency: str,
+            is_maker: bool,
+            trade_type: object,
+            amount: object,
+            price: object):
+        
+        gas_estimate: int = 130000  # approximate gas used for 0x market orders
+        transaction_cost_eth: double
 
         # there are no fees for makers on Radar Relay
-        if order_type is OrderType.LIMIT:
+        if is_maker:
             return TradeFee(percent=Decimal(0.0))
         # only fee for takers is gas cost of transaction
-        transaction_cost_eth = self._wallet.gas_price * gas_estimate / 1e18
+        transaction_cost_eth = RadarRelayMarket.class_wallet_object.gas_price * gas_estimate / 1e18
```

Comment:
`class_wallet_object` will not have value unless at least one of object of the market class is created, and we shouldn't access class member from static method. I think we need to get gas fee in a similar way we get fees for centralised exchanges.